### DISCUSSION TITLE
Rewind buttons don't work on Android

### DISCUSSION THREAD

**Discussion by Proserpinaa** [LunatiqueCoder/react-native-media-console#31] [Q&A] [answered]
Hello! 

I am working on a project, integrated a module for fullscreen video and using this package for controls. Everything seems to be fine, except that rewind forward/backwards doesn't work for me. 
I am using react-native-video v 5.2.1 and media-console v 2.0.1. 
I included rewindTime prop in my VideoPlayer component. 

Does anybody have any recommendation on what I can try to fix it? 

Thank you! Any help is highly appreciated!

**Comment by LunatiqueCoder** [comment, marked as answer]
Hello @Proserpinaa 

We can't seem to find any issue with the rewind buttons, could you tell us more about your problem? What is happening more precisely? Precise a code snippet to reproduce.

#### SUGGESTION:
The rewind button is based on the `seek` functionality from the `react-native-video`. There is a known issue that seek does not work correctly on Android: https://github.com/react-native-video/react-native-video/issues/2230#issuecomment-892982288

This bug should be fixed in `react-native-video@6.0.0-alpha.4`, you can try the alpha version and see how that works or you could use [`patch-package`](https://www.npmjs.com/package/patch-package) to apply this fix yourself.

I really hope this is your issue and you'll be able to fix it. 🙏 Let us know if you managed to find the issue and/or a fix.

**Reply by Proserpinaa** [reply]
Thank you! 

I tried the 6 alpha version and everything seems to work fine. 
I rebuild the component, switched some functions and everything seems fine.

**Reply by LunatiqueCoder** [reply]
@Proserpinaa  Happy to hear that! Keep in mind that it’s an apha version, that means it might not be stable and there might still be problems. In that case, be sure to open an issue at react-native-video (if there isn’t one open already). I’m sure they would be happy to help.
